utils/kata-manager.sh

```diff
@@ -680,11 +688,12 @@ configure_kata()
 handle_kata()
 {
 	local version="${1:-}"
+	local tarball="${2:-}"
 
 	local enable_debug="${2:-}"
 	[ -z "$enable_debug" ] && die "no enable debug value"
 
-	install_kata "$version" "$enable_debug"
+	install_kata "$version" "$tarball" "$enable_debug"
```

Comment:
We seem to have removed the debug arg, so:

```suggestion
	install_kata "$version" "$tarball"
```

Reply:
I've removed this one, but as part of a different commit, as it was not exactly related to this PR.